services/motion/builtin/builtin_test.go

```diff
@@ -2008,3 +1984,43 @@ func TestNewValidatedMotionCfg(t *testing.T) {
 		})
 	})
 }
+
+func pollUntilMOGSuccessOrError(
```

Comment:
We have this already in the nav builtin service and it looks like its copy pasted.  I'm wondering if it makes sense to DRY this up by taking this code block putting it in the motion package as a public helper function and then calling it from both sites.  It would exist as some sugar for the Go SDK only and since we are mainly concerned with developing in this SDK probably useful long term

Reply:
Also I dont think anything is MOG specific so I would prefer to strip that name out

Reply:
done